mava/environment_loop.py

```diff
@@ -102,12 +102,31 @@ def __init__(
             self.episode_returns.update({agent: generate_zeros_from_spec(spec)})
```

Comment:
if you can remove this comment line 637-638
`TODO (Ruan): Checking whether we are using Jax or TF here
 this should be removed once we deprecate TF.`

Reply:
And remove the if statement in line 639

Reply:
Thanks @OmaymaMahjoub! These are great points! I would prefer to address in a diff PR though since this one is quite large already and hard to review. Is it okay if we create a new issue and update these things in a new PR?

Reply:
Yeah we will make a new PR for this